dom.bs

```diff
@@ -971,7 +971,7 @@ when something has occurred.
  <li><dfn for="event listener">type</dfn> (a string)
  <li><dfn for="event listener">callback</dfn> (null or an {{EventListener}} object)
  <li><dfn for="event listener">capture</dfn> (a boolean, initially false)
- <li><dfn for="event listener">passive</dfn> (a boolean, initially false)
+ <li><dfn for="event listener">passive</dfn> (null or a boolean)
```

Comment:
I don't think the lack of a default works here. https://html.spec.whatwg.org/#activate-an-event-handler doesn't specify a value for it, for instance.